internal/api/core/server.go

```diff
@@ -52,6 +53,7 @@ func NewPersesAPI(serviceManager dependency.ServiceManager, cfg config.Config) e
 	apiEndpoints := []endpoint{
 		configendpoint.New(cfg),
 		migrateendpoint.New(cfg.Schemas),
+		validateendpoint.New(serviceManager.GetSchemas()),
```

Comment:
I think we should be consistent here so either have
```go
migrateendpoint.New(cfg.Schemas),
validateendpoint.New(cfg.Schemas),
```
or
```go
migrateendpoint.New(serviceManager.GetSchemas()),
validateendpoint.New(serviceManager.GetSchemas()),
```

Reply:
I don't get why we need consistency here. the Migrate endpoint doesn't use the service `schemas` while it is used by the Validate endpoints.

Reply:
Ah nevermind I thought serviceManager.GetSchemas() was returning the schemas..